app/src/main/kotlin/cz/covid19cz/erouska/ui/contacts/ContactsFragment.kt

```diff
@@ -41,14 +44,14 @@ class ContactsFragment : BaseFragment<FragmentPermissionssDisabledBinding, Conta
     }
 
     private fun openImportant() {
-        showWeb(viewModel.getImportantUrl())
+        showWeb(viewModel.getImportantUrl(), customTabHelper)
     }
 
     private fun openFaq() {
-        showWeb(viewModel.getFaqUrl())
+        showWeb(viewModel.getFaqUrl(), customTabHelper)
     }
 
     private fun goToWeb() {
-        showWeb(viewModel.getHomepageLink())
+        showWeb(viewModel.getHomepageLink(), customTabHelper)
```

Comment:
It would be actually nice to refactor this so that we call it rather like `customTabHelper.showWeb(link)` but I'd leave that for a separate PR.